index.html

```diff
@@ -25,7 +25,40 @@
                 <img class="menu__icon" src="./images/Header/icons/hamburger.svg" alt="Menu icon" />
             </div>
         </header>
-    </div>
+
+        <div class="hero container">
+            <div class="hero__content">
+                <div class="hero__text">
+                <h2 class="hero__welcome">Hello</h2>
+                <h1 class="hero__title">I’m <span class="bold">Sarah Jackson</span></h1>
+                <p class="hero__desc">A UX / UI designer, product designer, content creator and creative director.</p>
+                <p class="hero__desc2">Of course, there’s so much more to me than just a few fancy titles. Scroll down and get to know me,</p>
+                </div>
+
+                <div class="hero__image">
+                    <picture>
+                        <source srcset="./images/Hero/profilePicture/ProfileDesktop.svg" media="(min-width: 1024px)">
+                        <source srcset="./images/Hero/profilePicture/ProfileTablet.svg" media="(min-width: 760px)">
+                        <img class="avatar" src="./images/Hero/profilePicture/ProfileDesktop.svg" alt="Profile Picture">
+                    </picture>
+                </div>
+            </div>
+
+            <div class="hero__scroll">
+                <div class="social__items">
+                    <a class="link" href="https://facebook.com" target="_blank" rel="noreferrer"><img class="social__icon" src="./images/Hero/icons/facebook.svg" alt="facebook icon" /></a>
+                    <a class="link" href="https://linkedin.com" target="_blank" rel="noreferrer"><img class="social__icon" src="./images/Hero/icons/linkedin-logo.svg" alt="linkedin icon" /></a>
+                    <a class="link" href="https://behance.com" target="_blank" rel="noreferrer"><img class="social__icon" src="./images/Hero/icons/behance.svg" alt="behance icon" /></a>
+                    <a class="link" href="https://twitter.com" target="_blank" rel="noreferrer"><img class="social__icon" src="./images/Hero/icons/twitter.svg" alt="twitter icon" /></a>
+                    <a class="link" href="https://dribbble.com" target="_blank" rel="noreferrer"><img class="social__icon" src="./images/Hero/icons/dribbble.svg" alt="dribbble icon" /></a>
+                </div>
+                <div class="scroll">
+                    <p class="scroll__text">SCROLL</p>
+                    <img class="scroll__icon" src="./images/Hero/icons/right-arrow.svg" alt="arrow" />
+                </div>
```

Comment:
Here you have done everything properly, except there is a default margin-top on the `p` and it changes the look.